lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -49,7 +49,7 @@
 #include <mayaUsd/ufe/UsdSceneItem.h>
 
 #include <ufe/globalSelection.h>
-#if UFE_PREVIEW_VERSION_NUM >= 2027 // #ifdef UFE_V2_FEATURES_AVAILABLE
+#ifdef UFE_V2_FEATURES_AVAILABLE
```

Comment:
Most changes just like this - replacing anything that uses preview value 2030 or less with _#ifdef UFE_V2_FEATURES_AVAILABLE_